### PR TITLE
IoT Based Automated Billing System

### LINKED ISSUE #1077: GSSOC'24 - IoT Based - ESP32 and MFRC522 (RFID Tag and Reader) Based Billing System
I would like to add a project under IoT section. The project uses an ESP32 along with MFRC522 RFID Reader and Tags to perform billing of items. The results including the price, quantity and price are displayed on a local web server.
Hardware Components - ESP32, LCD Display, MFRC522 Reader and Tag, Buzzer and Push Button.
Software - Arduino IDE
Please assign this issue to me, I am a GSSOC contributor.

### PR DESCRIPTION
**Pull request by Vrisha213**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

yes

### Description

Add the code, video readme and image files for the project

Fixes #1077 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: IOT(Internet of Things)/Basic/ESP32_Billing_System/readme.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+8. Arduino IDE - Software
+
+## The circuit diagram is as shown
+[Image](Images/circuit.png)
+
+## The implementation diagram is as shown
+[Image](Images/setup.png)
+
+## Working of the System
+
```
Comment (Kushal997-das) [inline IOT(Internet of Things)/Basic/ESP32_Billing_System/readme.md:23]: I want image should display here instead of any link .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Vrisha213 Hello add your project [here](https://github.com/Vrisha213/Project-Guidance/blob/main/IOT(Internet%20of%20Things)/README.md)

**Comment by Vrisha213** [comment]
@Kushal997-das Hi, I've made the required changes. Please check.
Thank you

**Comment by Kushal997-das** [comment]
@Vrisha213 Do you have LinkedIn?

**Comment by Vrisha213** [comment]
Done @Kushal997-das

File: IOT(Internet of Things)/README.md [hunk 2]
```diff
@@ -25,7 +25,8 @@
...
 | 19. | [Arduino Traffic Light Simulator](https://github.com/Kushal997-das/Project-Guidance/tree/main/IOT(Internet%20of%20Things)/Basic/Arduino_TrafficLight_Simulator) | 20. | [Arduino UltraSonic Sensor](https://github.com/Kushal997-das/Project-Guidance/tree/main/IOT(Internet%20of%20Things)/Basic/Arduino_UltraSonic_Sensor) | 21. | [Multi Arduino Knight Rider Lights](https://github.com/Kushal997-das/Project-Guidance/tree/main/IOT(Internet%20of%20Things)/Basic/Multi%20Arduino%20Knight%20Rider%20Lights)
-| 22. | [Arduino_Line_Follower_Robot](https://github.com/deedGhost/Project-Guidance/tree/main/IOT(Internet%20of%20Things)/Basic/Arduino_Line_Follower_Robot)
+| 22. | [Arduino_Line_Follower_Robot](https://github.com/deedGhost/Project-Guidance/tree/main/IOT(Internet%20of%20Things)/Basic/Arduino_Line_Follower_Robot)| 23. | [ESP32_Billing_System](https://github.com/Vrisha213/Project-Guidance/tree/main/IOT(Internet%20of%20Things)/Basic/ESP32_Billing_System)
```
Comment (Kushal997-das) [inline IOT(Internet of Things)/README.md:28]: Change the project name `ESP32_Billing_System` to `Automated Billing System`

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Vrisha213 Reduce the size of image a bit.

**Comment by Vrisha213** [comment]
Done @Kushal997-das .

**Review by Kushal997-das — APPROVED** [review]

**Pull request merged**